tests/test_schema_editor_partitioning.py

```diff
@@ -275,3 +275,73 @@ def test_schema_editor_add_default_partition(method, key):
     schema_editor.delete_partition(model, "mypartition")
     table = db_introspection.get_partitioned_table(model._meta.db_table)
     assert len(table.partitions) == 0
+
+
+@pytest.mark.postgres_version(lt=140000)
```

Comment:
Regular `DETACH` should work since Postgres 10. Not sure if we want to support it since that version, but just wanted to clear it up.